server/svix-server/src/v1/endpoints/health.rs

```diff
@@ -67,16 +71,41 @@ pub struct HealthReport {
 
     queue: HealthStatus,
     cache: HealthStatus,
+    store: HealthStatus,
 }
 
 #[derive(Deserialize, Serialize, Debug, PartialEq)]
 struct HealthCheckCacheValue(());
-kv_def!(HealthCheckCacheKey, HealthCheckCacheValue);
+#[derive(Deserialize, Serialize, Debug, PartialEq)]
+struct HealthCheckStoreValue(());
+
+kv_def!(
+    HealthCheckCacheKey,
+    HealthCheckCacheValue,
+    "SVIX_CACHE_HEALTH_CHECK"
+);
+impl HealthCheckCacheKey {
+    pub fn new(id: &str) -> HealthCheckCacheKey {
+        HealthCheckCacheKey(format!("{}{}", Self::KEY_PREFIX, id))
+    }
+}
+
+kv_def!(
```

Comment:
Kinda sucks that you can reuse the same kv for both cache and store, so you're not accidentally protected against mistakes there. Can we maybe have two variants to this macro that just wrap this one with a newtype?